### PR TITLE
XDM Schema changes - createRequisitionList

### PR DESCRIPTION
**Pull request by NadiyaS**
Original ticket with request to add changes to xdm schema: https://jira.corp.adobe.com/browse/DINT-794

### CODE REVIEW

File: components/datatypes/requisitionlist.schema.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/requisitionlist",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Requisition List",
+  "type": "object",
+  "description": "Requisition List specific properties",
```
Comment (erikmarr) [inline components/datatypes/requisitionlist.schema.json:12]:
```suggestion
  "description": "Requisition List properties",
```

File: components/datatypes/requisitionlist.schema.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "definitions": {
+    "requisitionList": {
+      "properties": {
+        "xdm:ID": {
+          "title": "Requisition List ID",
+          "type": "string",
+          "description": "Unique identifier for the requisition list."
```
Comment (erikmarr) [inline components/datatypes/requisitionlist.schema.json:19]:
```suggestion
          "description": "Unique identifier of the requisition list."
```

File: components/datatypes/requisitionlist.schema.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        },
+        "xdm:name": {
+          "title": "Requisition List Name",
+          "type": "string",
+          "description": "Name of requisition list set by customer."
```
Comment (erikmarr) [inline components/datatypes/requisitionlist.schema.json:24]:
```suggestion
          "description": "Name of the requisition list specified by the customer."
```

File: components/datatypes/requisitionlist.schema.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        },
+        "xdm:description": {
+          "title": "Requisition List Description",
+          "type": "string",
+          "description": "Description of requisition list set by customer."
```
Comment (erikmarr) [inline components/datatypes/requisitionlist.schema.json:29]:
```suggestion
          "description": "Description of the requisition list specified by the customer."
```

**Review by erikmarr — COMMENTED** [review]
Minor editorial suggestions.

File: components/datatypes/marketing/commerce.schema.json [hunk 5]
```diff
@@ -119,6 +119,22 @@
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "A product has been purchased instantly, potentially skipping the cart or checkout.",
           "meta:descriptionId": "commerce##xdm:instantPurchase##description##9211"
+        },
+        "xdm:requisitionListOpens": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
```
Comment (benjaminkalk) [inline components/datatypes/marketing/commerce.schema.json:124]: Should we have `title` property for any of these or the `meta` properties similar to others in this schema?

Comment (NadiyaS) [reply components/datatypes/marketing/commerce.schema.json:124]: Good catch about the title. I'll add now, but meta has some unique numbers I am not aware of and seems like it is always added by xdm team during the release.

**Review by benjaminkalk — APPROVED** [review]

**Review by erikmarr — APPROVED** [review]

**Review by mjasrotia — APPROVED** [review]

**Pull request merged**